plugins/continuous-toolbox/src/ContinuousToolbox.js

```diff
@@ -34,7 +34,10 @@ export class ContinuousToolbox extends Blockly.Toolbox {
 
         // Handle cutom categories (e.g. variables and functions)
         if (typeof itemContents === 'string') {
-          itemContents = {custom: itemContents};
+          itemContents = {
+            custom: itemContents,
+            kind: 'CATEGORY',
+          };
         }
         contents = contents.concat(itemContents);
```

Comment:
I might make this entire section from `let contents = []` to here a function so that if someone wants to handle this differently they can easily override that and return their own list of contents.

Reply:
good idea, added a function